contracts/token/ERC20/ERC20Basic.sol

```diff
@@ -7,7 +7,7 @@ pragma solidity ^0.4.18;
  * @dev see https://github.com/ethereum/EIPs/issues/179
  */
 contract ERC20Basic {
-  uint256 public totalSupply;
+  function totalSupply() public view returns (uint256);
```

Comment:
I got a compilation error because of this. It tells the following:
```
zeppelin-solidity/contracts/token/ERC20/ERC20Basic.sol:10:3: TypeError: Redeclaring an already implemented function as abstract
  function totalSupply() public view returns (uint256);
  ^---------------------------------------------------^
,zeppelin-solidity/contracts/token/ERC20/ERC20Basic.sol:10:3: TypeError: Redeclaring an already implemented function as abstract
  function totalSupply() public view returns (uint256);
  ^---------------------------------------------------^
```

Reply:
Where did that happen @hlogeon? If you can reproduce would you mind opening a new issue please?